src/Xamarin.Android.Build.Tasks/Tasks/RuntimeIdentifierToAbi.cs

```diff
@@ -14,17 +14,21 @@ public class RuntimeIdentifierToAbi : AndroidTask
 
 		public string RuntimeIdentifier { get; set; }
 
+		public string [] RuntimeIdentifiers { get; set; }
+
 		[Output]
 		public string SupportedAbis { get; set; }
 
 		public override bool RunTask ()
 		{
-			if (!string.IsNullOrEmpty (RuntimeIdentifier)) {
+			if (RuntimeIdentifiers != null && RuntimeIdentifiers.Length > 0) {
```

Comment:
Implicit to this is that `$(RuntimeIdentifiers)` takes *precedence* over `$(AndroidSupportedAbis)`.  Is this desirable/intentional?  What will be the fallout to this decision?

For example, if I have an existing `App.csproj` which overrides `$(AndroidSupportedAbis)`, and then I migrate to .NET 5/6 and *don't* touch `$(RuntimeIdentifiers)`… What Happens?

Given that this PR always provides a default `$(RuntimeIdentifiers)` value, what I *think* will happen is that we'll effectively *ignore* `$(AndroidSupportedAbis)`, replacing the "original overridden value" with the defaults that *we* provide.

Is this useful or desirable?  (It might be!  I don't know!  I *am* wondering if there's been any consideration here, particularly as the commit message doesn't mention this semantic.)

Meanwhile, is the *opposite* possible?  If `$(AndroidSupportedAbis)` is overridden, and `$(RuntimeIdentifiers)` *isn't* -- though making this determination may itself be difficult! -- would it be at all possible to populate `$(RuntimeIdentifiers)` based on the value of `$(AndroidSupportedAbis)`?

Reply:
Eventually, I think that `RuntimeIdentifier` and/or `RuntimeIdentifiers` will be implicitly set via:

    <TargetFrameworkVersion>net5.0-android</TargetFrameworkVersion>

And so, it seems like `$(AndroidSupportedAbis)` would always be ignored going forward? Is it worth trying to support both?

Either way, I should document something about `$(RuntimeIdentifiers)` being preferred in `DotNet5.md`.

Reply:
If `$(AndroidSupportedAbis)` will be *ignored* going forward, then we should consider adding a warning if the property is set at all, as having something set which doesn't do anything will be a source of confusion.

This may complicate migrating from "long-form" to "short-form" projects, i.e. it'll be a property that needs to be *removed* as part of the migration, but this may be for the best.

Alternatively, we would need to ensure that if `$(AndroidSupportedAbis)` is set, the values within it *contribute to* `$(RuntimeIdentifiers)`, so that when it's set, the specified ABIs will still appear in the resulting `.apk`/`.aab`.

Reply:
Since `$(RuntimeIdentifiers)` is needed for `Restore` to even work... I think we really should require it over `$(AndroidSupportedAbis)`.

I added `XA0036` if `$(AndroidSupportedAbis)` is present, @brendanzagaeski you might review what I added for this, thanks.